### PR TITLE
Added AzureLogin cmdlets

### PR DESCRIPTION
**Pull request by hovsepm**
#102920948

### CODE REVIEW

**Comment by stankovski** [comment]
@azuresdkci retest this please

File: src/Common/Commands.ResourceManager.Profile/LoginAzureRMAccount.cs [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+                azureAccount.Type = AzureAccount.AccountType.AccessToken;
+            }
+            else if (ServicePrincipal.IsPresent)
+            {
+                azureAccount.Type = AzureAccount.AccountType.ServicePrincipal;
+            }
+            else
+            {
+                azureAccount.Type = AzureAccount.AccountType.User;
+
```
Comment (markcowl) [inline src/Common/Commands.ResourceManager.Profile/LoginAzureRMAccount.cs]: remove extra space

File: src/Common/Commands.ResourceManager.Profile/LoginAzureRMAccount.cs [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+                azureAccount.SetProperty(AzureAccount.Property.Tenants, new[] { Tenant });
+            }
+
+            if( AzureRMCmdlet.DefaultProfile == null)
+            {
+                AzureRMCmdlet.DefaultProfile = new AzureRMProfile();
+            }
+
+            var profileClient = new RMProfileClient(AzureRMCmdlet.DefaultProfile);
+
```
Comment (markcowl) [inline src/Common/Commands.ResourceManager.Profile/LoginAzureRMAccount.cs]: remove extra space

**Comment by markcowl** [comment]
Two minor nits.  Ship It!

**Pull request merged**